test/templates/helpers/templateMongooseHelpers.test.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  eq,
+  getMongooseSystemType,
+  isMongooseSchemaFunctionRequired,
+  isMongooseSchemaFunctionIdRequired,
+} from "../../../src/templates/helpers/templateHelpers";
+
+describe("templateMongooseHelpers", () => {
+  describe("getMongoosePrimitive()", () => {
+    test("Should return equivalent mongoose compatible primitive", () => {
+      expect(getMongooseSystemType("base64Binary")).toEqual("String");
+      expect(getMongooseSystemType("boolean")).toEqual("Boolean");
+      expect(getMongooseSystemType("canonical")).toEqual("String");
+      expect(getMongooseSystemType("code")).toEqual("String");
+      expect(getMongooseSystemType("date")).toEqual("Date");
+      expect(getMongooseSystemType("dateTime")).toEqual("Date");
+      expect(getMongooseSystemType("decimal")).toEqual("Number");
+      expect(getMongooseSystemType("id")).toEqual("String");
+      expect(getMongooseSystemType("instant")).toEqual("Date");
+      expect(getMongooseSystemType("integer")).toEqual("Number");
+      expect(getMongooseSystemType("markdown")).toEqual("String");
+      expect(getMongooseSystemType("oid")).toEqual("String");
+      expect(getMongooseSystemType("positiveInt")).toEqual("Number");
+      expect(getMongooseSystemType("question")).toEqual("String");
+      expect(getMongooseSystemType("string")).toEqual("String");
+      expect(getMongooseSystemType("time")).toEqual("Date");
+      expect(getMongooseSystemType("unsignedInt")).toEqual("Number");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5966907)